resources/keepalived-docker/manage-keepalived.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export assignedIP="$PROVISIONING_IP/$PROVISIONING_CIDR"
+export assignedIP="$PROVISIONING_IP"
```

Comment:
change to $PROVISIONING_IP/32 I guess

Reply:
We need to wait for @maelk on this, he knows the actual bug and his suggestion was to remove it.

Reply:
Ah sorry you are right @namnx228

Reply:
Done